### PR TITLE
Products & Categories

### PR DESCRIPTION
**Pull request by AshutoshM29**
Raised a small PR for product mapping and categories for the e-com website

Files to check:
[Products](src/backend/db/products.js)
[Categories](src/backend/db/categories.js)
[Card](src/components/Card/Card-Vertical/Card-Vertical.jsx)
[Product.jsx](src/hooks/useProducts.jsx)

Files to avoid: 
BIN files
[Index](public/index.html)
[App-CSS](src/App.css)

### CODE REVIEW

File: src/components/Card/Card-Vertical/Card-Vertical.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Link } from "react-router-dom";
+import { useProducts } from "../../../hooks/useProducts";
+import "../../../pages/Product/product.css"
+const Cards=()=>{
+const {products}=useProducts();
+
+return ( <div className="container-card">
+    { products.map((products)=>{
```
Comment (Sahil25061999) [inline src/components/Card/Card-Vertical/Card-Vertical.jsx:8]: Just a small suggestion give different variable name for the map

**Review by Sahil25061999 — APPROVED** [review]
Mostly looks good there are no major changes required, just suggested a small change

File: src/pages/Homepage/Homepage.jsx [hunk 2]
```diff
@@ -2,113 +2,80 @@ import { Link } from 'react-router-dom'
...
-                <img className="img-category" src={catClothing} alt="Clothing & Accessories" />
-                <p className="desc-category">Clothing & Accessories</p>
+            <img className="img-category" src={catClothing} alt="Clothing & Accessories" />
+            <p className="desc-category">Clothing & Accessories</p>
             </Link>
 
             <Link to="/Product">
-                <img className="img-category" src={catTreat} alt="treat" />
-                <p className="desc-category">Treat</p>
+            <img className="img-category" src={catTreat} alt="treat" />
```
Comment (rajaranjanmr) [inline src/pages/Homepage/Homepage.jsx:57]: suggestion : try to use , map for plotting data

**Review by rajaranjanmr — APPROVED** [review]
rest seems to be good and can be merged

**Pull request merged**